cache/azblobproxy/azblobproxy.go

```diff
@@ -82,6 +82,12 @@ func (c *azBlobCache) Get(ctx context.Context, kind cache.EntryKind, hash string
 
 	resp, err := client.Download(context.Background(), nil)
 	if err != nil {
+		var stgErr *azblob.StorageError
+		if errors.As(err, &stgErr) && stgErr.ErrorCode == azblob.StorageErrorCodeBlobNotFound {
+			cacheMisses.Inc()
```

Comment:
Since we do `cacheMisses.Inc()` in both paths, maybe do that as the first line inside the `if err != nil {` block?